i18n/hr.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
@ctlaltdieliet we have deleted this file in a commit late last week

Reply:
yeah, I thought I removed it in the PR as well but it seems I didn't. (or didn't commit it). It removed in webapp as well